src/web/client/common/portalSchema.ts

```diff
@@ -1311,9 +1311,11 @@ export const portal_schema_data = {
                 "_disableplugins": "true",
                 "_foldername": "content-snippets",
                 "_propextension": "contentsnippet",
-                "_exporttype": "SubFolders",
+                "_exporttype": "SingleFolder",
```

Comment:
@tyaginidhi I think this change we will have to revert. Let's discuss about this. I think we are missing language information for the content snippets?

Reply:
Lets make the content snippet language change as part of next PR specific to content snippet